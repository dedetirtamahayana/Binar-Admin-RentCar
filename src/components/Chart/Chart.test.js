// import renderer from 'react-test-renderer';
// import Chart from './Chart.js';
// import {describe,test} from '@jest/globals';
// import { BrowserRouter } from 'react-router-dom';
// import { Provider } from 'react-redux';
// import {Store} from "../../store";
// describe('Chart.js should work as expected', ()=>{
//   test('Chart.js should match with snapshot',()=>{
//     const comp = renderer
//     .create(
//      <BrowserRouter>
//       <Provider store={Store}>
//         <Chart/>
//       </Provider>
//      </BrowserRouter>
//       )
//     .toJSON();

//     expect (comp).toMatchSnapshot();
//   });
// });