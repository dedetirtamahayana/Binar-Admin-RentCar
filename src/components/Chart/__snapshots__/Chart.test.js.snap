// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart.js should work as expected Chart.js should match with snapshot 1`] = `
<div
  className="container chart-section"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <img
        className="text-img "
        src="title-square.png"
      />
      <p
        className="fw-bold title-img"
      >
        Rented Car Data Visualization
      </p>
      <div
        className="detailSelect col"
      >
        Month
      </div>
      <form
        className="d-flex mb-5"
      >
        <select
          aria-label="Default select example"
          className="form-select"
          onChange={[Function]}
          style={
            Object {
              "borderRadius": 2,
              "fontSize": 12,
              "width": 150,
            }
          }
        >
          <option
            disabled={true}
          >
            Open this select menu
          </option>
          <option
            value={1}
          >
            January
             - 2022
          </option>
          <option
            value={2}
          >
            February
             - 2022
          </option>
          <option
            value={3}
          >
            March
             - 2022
          </option>
          <option
            value={4}
          >
            April
             - 2022
          </option>
          <option
            value={5}
          >
            May
             - 2022
          </option>
          <option
            value={6}
          >
            June
             - 2022
          </option>
          <option
            value={7}
          >
            July
             - 2022
          </option>
          <option
            value={8}
          >
            August
             - 2022
          </option>
          <option
            value={9}
          >
            September
             - 2022
          </option>
          <option
            value={10}
          >
            October
             - 2022
          </option>
          <option
            value={11}
          >
            November
             - 2022
          </option>
          <option
            value={12}
          >
            December
             - 2022
          </option>
        </select>
        <br />
        <button
          className="btnGo btn btn-outline-none"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Go
        </button>
      </form>
    </div>
    <div
      className="mixed-chart"
    >
      <div
        className="chart-mobil"
      />
    </div>
  </div>
</div>
`;
